packages/block-editor/src/components/block-mover/mover-description.native.js

```diff
@@ -13,10 +13,8 @@ const horizontalMover = {
 	forwardButtonIcon: arrowRight,
 	backwardButtonHint: __( 'Double tap to move the block to the left' ),
 	forwardButtonHint: __( 'Double tap to move the block to the right' ),
-	/* translators: %s: block title e.g: "Paragraph" */
-	firstBlockTitle: __( 'Move %s Left' ),
-	/* translators: %s: block title e.g: "Paragraph" */
-	lastBlockTitle: __( 'Move %s Right' ),
+	firstBlockTitle: __( 'Move block left' ),
```

Comment:
It seems like something that should be done in a separate PR. WDYT? I'm rather a fan of small PRs w/o side tasks

Reply:
Ok, will move all the changes related to block mover to the separated PR ✌️